README.md

```diff
@@ -43,6 +43,35 @@ reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --
 reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --insert-rules --trigger-date-time=<DATE_TIME> --max-cases=1000000 
 ```
 
+## Response-driven reminder scheduling
+We can create response-driven action rules from a provided CSV file of LSOAs from a bucket. You will need to copy this file from the bucket to your cloud project, then copy to the running toolbox instance.
```

Comment:
Should be "cloud shell" rather than "cloud project"? I guess this is just a high level idea of how it'll be used, the detailed instructions will be runbook-ed so not a critical comment.

Reply:
Follow up, best to make sure a ticket is on the board now for writing the runbook for this